src/libraries/Microsoft.Extensions.Configuration.Ini/src/IniConfigurationExtensions.cs

```diff
@@ -31,7 +31,7 @@ public static IConfigurationBuilder AddIniFile(this IConfigurationBuilder builde
         /// <param name="builder">The <see cref="IConfigurationBuilder"/> to add to.</param>
         /// <param name="path">The path relative to the base path stored in
         /// <see cref="IConfigurationBuilder.Properties"/> of <paramref name="builder"/>.</param>
-        /// <param name="optional">Whether the file is optional.</param>
+        /// <param name="optional"><see langword="true"> if the file is optional; otherwise, <see langword="false"/> .</param>
```

Comment:
```suggestion
        /// <param name="optional"><see langword="true"/> if the file is optional; otherwise, <see langword="false"/> .</param>
```

Missing close `/` in the see-true